documentation/asciidoc/computers/camera/troubleshooting.adoc

```diff
@@ -0,0 +1,16 @@
+== Troubleshooting
+
+If your Camera Module doesn't work like you expect, try some of the following fixes:
+
+* On Raspberry Pi 3 and earlier devices running Raspberry Pi OS _Bullseye_ or earlier:
+** To enable hardware-accelerated camera previews, enable *Glamor*. To enable Glamor, enter `sudo raspi-config` in a terminal, select `Advanced Options` > `Glamor` > `Yes`. Then reboot your Raspberry Pi with `sudo reboot`.
+** If you see an error related to the display driver, add `dtoverlay=vc4-fkms-v3d` or `dtoverlay=vc4-kms-v3d` to `/boot/config.txt`. Then reboot your Raspberry Pi with `sudo reboot`.
+** The graphics hardware can only support images up to 2048×2048 pixels, which places a limit on the camera images that can be resized into the preview window. As a result, video encoding of images larger than 2048 pixels across produces corrupted or missing preview images.
+* On Raspberry Pi 4, the graphics hardware can only support images up to 4096×4096 pixels, which places a limit on the camera images that can be resized into the preview window. As a result, video encoding of images larger than 4096 pixels across produces corrupted or missing preview images.
+* The preview window may show display tearing in a desktop environment. This is a known, unfixable issue.
+* Check that the FFC (Flat Flexible Cable) is firmly seated, fully inserted, and that the contacts face the correct direction. The FFC should be evenly inserted, not angled.
+* If you use a connector between the camera and your Raspberry Pi, check that the ports on the connector are firmly seated, fully inserted, and that the contacts face the correct direction.
+* Check to make sure that the FFC (Flat Flexible Cable) is attached to the CSI (Camera Serial Interface), _not_ the DSI (Display Serial Interface). The connector fits into either port, but only the CSI port powers and controls the camera. Look for the `CSI` label printed on the board near the port.
+* Run `sudo apt update` and `sudo apt full-upgrade` to ensure that you're running the latest software.
```

Comment:
This should probably just link out to our standard "updating" section of the docs?